etcd/docker/running.sls

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+{% from "etcd/map.jinja" import etcd with context -%}
+
+   {% if etcd.docker.stop_etcd_service_first %}
+include:
+  - etcd.service.stopped
+   {% endif %}
+
+etcd-docker-python-module:
```

Comment:
Would love to see this changed to package and as a fallback pip...

Reply:
Great suggestion @aboe76 . I checked and package names in Debian / Fedora / Archlinux are `python-docker, python3-docker`. The Centos extras-repo includes `python-docker-py`.

Now you made this suggestion, I propose removing pip.installed completely and just using package instead. If someone has problem in future they can raise PR on this.

Reply:
I pushed commit to fix conflicts.

Reply:
@noelmcloughlin I think removing pip here is no problem, because you are already relying on "docker-formula", the only thing is to enable ```docker.install_docker_py``` variable...so the docker-formula installs the depency.

Reply:
I decided to use `onfail_in` and that works nicely.  I'll leave pip.installed there! thx